polaris-service/src/test/java/io/polaris/service/test/SnowmanCredentialsExtension.java

```diff
@@ -178,29 +178,29 @@ public void afterAll(ExtensionContext extensionContext) throws Exception {
             adminSecrets.getMainSecret(),
             realm);
 
-    try (Response deletePrResponse =
```

Comment:
Will this still close if an error is thrown? I think this one maybe should stay as a try-finally

Reply:
yes.
try-finally with empty try block is equivalent to calling close directly.

Reply:
Correct. try-with-resources is "just" a shorthand for one or more auto-closeables.
```java
AutoCloseable c1 = openSomething();
try {
        AutoCloseable c2 = openSomething();
        try {
                doSomethingWithC1andC2();
        } finally {
                c2.close();
        }
} finally {
        c1.close();
}
```